src/main/java/com/conveyal/gtfs/graphql/GraphQLGtfsSchema.java

```diff
@@ -644,6 +651,20 @@
                     // DataFetchers can either be class instances implementing the interface, or a static function reference
                     .dataFetcher(new JDBCFetcher("patterns"))
                     .build())
+            .field(newFieldDefinition()
+                .name("shapes")
+                .type(new GraphQLList(shapeType))
+                .argument(intArg(ID_ARG))
+                .argument(intArg(LIMIT_ARG))
+                .argument(intArg(OFFSET_ARG))
+                .argument(floatArg(MIN_LAT))
+                .argument(floatArg(MIN_LON))
+                .argument(floatArg(MAX_LAT))
+                .argument(floatArg(MAX_LON))
+                .argument(multiStringArg("shape_id"))
```

Comment:
Do any of these get used in the PolylineFetcher? If not, they should be removed.